docs/sdks/client-sdks/flutter/quickstart.mdx

```diff
@@ -0,0 +1,290 @@
+---
+title: Quickstart
+sidebar_position: 2
+---
+
+The Eppo SDK enables feature flags and experiments in your Dart and Flutter applications with only a few lines of code.
+
+The SDK handles all the complexity of feature flag evaluation and experiment assignment locally in your application. 
+This guide will walk you through installing the SDK and implementing your first feature flag, experiment, and contextual bandit.
+
+## Installation
+
+First, add the SDK to your `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  eppo: ^1.0.0
+```
+
+If using Dart, run:
+
+```shell
+dart pub get
+```
+
+If using Flutter, run:
+
+```shell
+flutter pub get
+```
+
+## Feature Flags
+
+Feature flags are a way to toggle features on and off without needing to deploy code.
+
+### Initialize the SDK
+
+[Create an SDK key](/sdks/sdk-keys) if you don't already have one.
+
+```dart
+import 'package:eppo/eppo.dart';
+
+// Create subject
+final subject = Subject(
+  subjectKey: subjectKey,
+  subjectAttributes: ContextAttributes(),
+);
+
+// Create precompute arguments
+final precomputeArgs = PrecomputeArguments(subject: subject);
+
+// Create SDK options
+final sdkOptions = SdkOptions(
+  sdkKey: sdkKey
+);
+
+// Create client
+Eppo.initialize(precomputeArgs, sdkOptions);
+```
+
+:::note
+The SDK key is different from the project API key. You can find your SDK key in the [SDK Keys section of the Eppo interface](https://eppo.cloud/configuration/environments/keys).
+:::
+
+### How assignments work
```

Comment:
note to self - tell people to add a hook on flutter app open to re-initialize the SDK